examples/react-native/src/ui/components/authenticator/reset-password/Example.tsx

```diff
@@ -3,15 +3,18 @@ import { StyleSheet, View } from 'react-native';
 
 import { Authenticator } from '@aws-amplify/ui-react-native';
 import { Amplify } from 'aws-amplify';
+import { parseAWSExports } from '@aws-amplify/core/lib-esm/parseAWSExports';
 
 import { SignOutButton } from '../SignOutButton';
 import awsconfig from './aws-exports';
 
-Amplify.configure({
-  ...awsconfig,
-  // mock server endpoint for Detox e2es
-  Auth: { endpoint: 'http://127.0.0.1:9091/' },
-});
+const config = parseAWSExports(awsconfig);
+if (!config.Auth) {
+  throw new Error('Auth config not found');
+}
+// mock server endpoint for Detox e2es
+config.Auth.Cognito.userPoolEndpoint = 'http://127.0.0.1:9091/';
```

Comment:
Thinking we make a util for this and add an .env variable for USER_POOL_END_POINT so we can use these examples for manual testing

Reply:
Would love to pick your brain on this since I can't get other env like EXAMPLE APP variables to work when testing locally.